src/Sulu/Bundle/WebsiteBundle/Controller/DefaultController.php

```diff
@@ -41,44 +40,4 @@ public function indexAction(StructureInterface $structure, $preview = false, $pa
 
         return $response;
     }
-
-    /**
-     * Creates a redirect for configured webspaces.
-     *
-     * @param Request $request
-     *
-     * @return Response
-     *
-     * @deprecated since 1.2 use SuluWebsiteBundle:Redirect:redirectWebspace instead
```

Comment:
yes both of them (: the `$this->forward` will do something different which is quite difficult to solve in unit tests without "version" flag. I thought about that we should use this version to "collect" smaller upgrades which doesn't hurt which we has already done in develop.

See:
* https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php#L64
* https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php#L68